bpf/bpf_lxc.c

```diff
@@ -835,8 +835,7 @@ ipv6_policy(struct __sk_buff *skb, int ifindex, __u32 src_label, __u8 *reason)
 	// Do not redirect again if the packet is coming from the egress proxy.
 	if ((ret == CT_REPLY || ret == CT_RELATED) && ct_state.proxy_redirect &&
 	    !tc_index_skip_egress_proxy(skb)) {
-		// Stack will do a socket match and deliver locally
-		return skb_redirect_to_proxy(skb, 0);
+		return TC_ACT_PROXY_REDIRECT;
```

Comment:
We should make this the same as the IPv4:
* Same trace message
* It's also redirecting with `*proxy_port` set as 0, which should be called out either in code or in comment